www-servers/automx2/automx2-2019.0.ebuild

```diff
@@ -23,6 +24,10 @@ RDEPEND="${BDEPEND}"
 
 python_prepare_all() {
 	sed -i -e "/('scripts'/d" setup.py || die
+	if use doc; then
```

Comment:
This if block should go into the appropriate *_install phase. Not in the prepare phase.

Reply:
Moved. I thought that variables like `DOCS`, which can be defined in global scope, should be set as early as possible.

Reply:
Yes, that is correct, but as they are global you can do it outside of a particular phase, but it will be called from the respective *_install phase.

Reply:
PMS does state however that global variables shouldn't be modified within these phases. That's why we usually use `local` inside a phase.

Also why are these in string form and not array? PM will have to do conversion now.

Reply:
The [developer manual](https://devmanual.gentoo.org/ebuild-writing/variables/) states that DOCS and HTML_DOCS define "An array or space-delimited list of documentation files".

Reply:
@juippis Do you have a suggestion for when and how to define the two variables?

Reply:
changing `DOCS` and `HTML_DOCS` is fine, even if they're global variables. But please make them arrays, as this is more customary and less smelly.